services/player-service/domain/src/main/java/com/kjeldsen/player/domain/PlayerActualSkills.java

```diff
@@ -0,0 +1,50 @@
+package com.kjeldsen.player.domain;
+
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public record PlayerActualSkills(Map<PlayerSkill, Integer> values) {
```

Comment:
PlayerActualSkills -> PlayerSkills

I do not think Actual in the name is giving us any extra information, but confusing us in the future. Let's keep it generic